pkgs/development/tools/hindsight/default.nix

```diff
@@ -0,0 +1,66 @@
+{ pkgs, lib, fetchFromGitHub }:
+
+with pkgs.python3Packages;
+
+buildPythonApplication rec {
+  pname = "hindsight";
+  version = "2021.01.16";
+
+  src = fetchFromGitHub {
+    owner = "obsidianforensics";
+    repo = pname;
+    rev = version;
+    sha256 = "0ibvwm36lfqfrm8h542v9fdf74czbf6rfajrcdmk1gwxv4d5nxa7";
+  };
+
+  nativeBuildInputs = with pkgs.nodePackages; [
+    esinstall
+    pkgs.nodejs
+    sqlite-view
+    terser
+  ];
+
+  propagatedBuildInputs = [
+    bottle
+    keyring
+    puremagic
+    pycryptodomex
+    pytz
+    XlsxWriter
+  ];
+
+  postPatch = ''
+    patchShebangs ./install-js.sh
+
+    # Symlink dependencies, per advice: https://nodejs.org/api/esm.html#esm_no_node_path.
+    mkdir -p node_modules
+    _IFS=$IFS
+    IFS=:
+    for path in $NODE_PATH; do
+      ln -s $path/* node_modules/
+    done
+    IFS=$_IFS
+
+    # Don't exclude the javascript assets we're building.
+    sed -i '/web_modules/d' MANIFEST.in
```

Comment:
Please use substituteInPlace or a patch to notice changes. sed fails silently.

Reply:
Is there a way to replace the whole line if there's a match with substituteInPlace?

Reply:
You can input the entire line but not by regex. Using sed when requiring regex is okay.